readthedocs/api/v2/serializers.py

```diff
@@ -162,11 +203,17 @@ class BuildAdminSerializer(BuildSerializer):
 
     """Build serializer for display to admin users and build instances."""
 
+    commands = BuildCommandSerializer(many=True, read_only=True)
+
     class Meta(BuildSerializer.Meta):
         # `_config` should be excluded to avoid conflicts with `config`
         exclude = ('_config',)
 
 
+class BuildAdminUISerializer(BuildAdminSerializer):
+    commands = BuildCommandUISerializer(many=True, read_only=True)
```

Comment:
This needs a docstring, I'm pretty confused about all the different serializers.